components/feature/contextmenu/src/main/java/mozilla/components/feature/contextmenu/DefaultSelectionActionDelegate.kt

```diff
@@ -68,28 +69,31 @@ class DefaultSelectionActionDelegate(
         else -> null
     }
 
-    override fun performAction(id: String, selectedText: String): Boolean = when (id) {
-        SEARCH -> {
-            searchAdapter.sendSearch(false, selectedText)
-            true
+    override fun performAction(id: String, selectedText: String): Boolean {
+        emitTextSelectionClickFact(id)
```

Comment:
Looks good! Let's add a simple test that checks `performAction` always emits a fact so we don't mistakenly remove this call.